api/domain/commanddata/cluster.proto

```diff
@@ -27,21 +28,43 @@ package commanddata;
 message CreateCluster {
   // Unique name of the cluster, to be utilized for generating unique labels
   // and symbols, e.g. with metrics.
-  string name = 1;
+  string name = 1 [(validate.rules).string = {pattern: "^[a-zA-Z][A-Za-z0-9_-]+$", max_bytes: 60}];
   // Display name of the cluster
-  string display_name = 2;
+  string display_name = 2 [(validate.rules).string.max_bytes = 150];
   // DNS name or IP address of the clusters KubeAPIServer
-  string api_server_address = 3;
+  string api_server_address = 3 [(validate.rules).string.pattern = "^(https?://)?[^\\s/$.?#/_].[^\\s_]*$"];
   // Bundle of CA certificates of the cluster, PEM encoded
   bytes ca_cert_bundle = 4;
+  // not supported yet
```

Comment:
What do you mean with not supported yet? By the library we're using?

Reply:
yes by the library. Take a look at this example (this one is supported)
string x = 1 [(validate.rules).string = {in: ["foo", "bar", "baz"]}];

i was hoping something similar is also supported for bytes prefix/suffix but apparently not yet.

Reply:
i kept the comment as a reminder for the future, should I remove it?

Reply:
Ah, no keep it. It's fine.